package.json

```diff
@@ -11,9 +12,15 @@
     "test": "mocha 'test/**/*.spec.js'",
     "test:all-schemas": "mocha 'test/common/all-schemas.spec.js'",
     "watch": "node src/start watch",
+    "link": "yarn link:vets-api; yarn link:vets-website",
+    "link:vets-api": "cd ../vets-api; bundle config set local.vets_json_schema ../vets-json-schema; bundle config disable_local_branch_check true",
+    "link:vets-website": "yarn link; cd ../vets-website; yarn link vets-json-schema",
```

Comment:
should move these to a shell script, to keep the package.json cleaner and can annotate the commands